pages/home/tasks.js

```diff
@@ -155,23 +172,31 @@ export default function Home() {
               );
           })}
 
+        {/*-- Mapped tasks for parents without reviewed tasks--*/}
+
         {user.type === "Parent" &&
           tasks.map(task => {
             if (task.review !== "in review" && task.review !== "reviewed") {
               return (
                 <StyledListElements key={task._id}>
                   <h2>{task.title}</h2>
-                  <StyledImage
+                  <Image
                     priority={true}
```

Comment:
You could also write it like this:

```suggestion
                    priority
```